site/src/components/WorkspaceBuildProgress/WorkspaceBuildProgress.tsx

```diff
@@ -0,0 +1,94 @@
+import LinearProgress from "@material-ui/core/LinearProgress"
```

Comment:
We should add stories here. I could easily see something accidentally happening and this not looking nice.

Reply:
It's covered in the Workspace Starting state story.

Reply:
Reasonable